pyproject.toml

```
@@ -6,3 +6,17 @@ requires = [
     "pybind11>=2.6.1",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.cibuildwheel]
+skip = "pp*"
+test-extras = "test"
+test-command = "pytest {project}/tests"
+manylinux-x86_64-image = "manylinux_2_24"
+manylinux-i686-image = "manylinux_2_24"
+
+[tool.cibuildwheel.linux]
+before-all = [
+    "apt-get update -y",
+    "apt-get install -y libmpfr-dev libboost-dev",
+]
+repair-wheel-command = "export LD_LIBRARY_PATH=$PWD/_fastjet_core "
```

```suggestion
repair-wheel-command = [
    "export LD_LIBRARY_PATH=$PWD/_fastjet_core",
    "auditwheel repair -w {dest_dir} {wheel}",
]
```

You need to include the original command, as well - this replaces the command.

---

```suggestion
repair-wheel-command = "LD_LIBRARY_PATH=$PWD/_fastjet_core auditwheel repair -w {dest_dir} {wheel}"
```

This is probably fine, as well

---

Fixed it in the latest commit.